x-pack/plugins/security_solution/public/flyout/shared/components/expandable_panel.tsx

```diff
@@ -88,7 +88,7 @@ export const ExpandablePanel: React.FC<ExpandablePanelPanelProps> = ({
     expandable: false,
     expandedOnFirstRender: false,
   },
-  'data-test-subj': dataTestSubj,
+  'data-test-subj': dataTestSubj = 'expandablePanel',
```

Comment:
`data-test-subj` is optional, so it is better to define a default value instead of rendering `undefined`.

Reply:
@machadoum 

hmmmm I'm actually not sure this is better. While I understand that having undefined isn't a really good solution either, having a default value on a reusable component means that we will end up with potentially many many components in the DOM with the same `data-test-subj`. Yes is was the case with `undefined`, but now the value actually exists and developers might be tempted to use it, which could end up with multiple hits.

Following good practices, `data-test-subj` values should always be unique.

Would there be a way to conditionally set the property if it's value exists instead?

Reply:
if it's not easy to do the condition, I would revert that change and keep `undefined` for now until we find a better solution. We already have those optional `data-test-subj` in a bunch of places, so it would make sense to tackle all of them at the same time with a good fix...
Thoughts?

Reply:
Ok. I will revert the change.